client/src/components/Weather/index.js

```diff
@@ -0,0 +1,134 @@
+import React, { useState, useEffect } from 'react';
+import ReactDOM from 'react-dom';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faCloud } from '@fortawesome/free-solid-svg-icons';
+import { faCloudRain } from '@fortawesome/free-solid-svg-icons';
+import { faBolt } from '@fortawesome/free-solid-svg-icons';
+import { faSnowflake } from '@fortawesome/free-solid-svg-icons';
+import { faSun } from '@fortawesome/free-solid-svg-icons';
+import { faSmog } from '@fortawesome/free-solid-svg-icons';
+import './style.css';
+
+const API_KEY = "8e20c3b423d1a8139959af74dcb9cba2";
```

Comment:
For security reasons, you should be storing the API key in the code itself. You should store it elsewhere like a text file or something that can be read into the code but isn't uploaded onto git so no-one knows your personal API key.